client/ayon_core/addon/interfaces.py

```diff
@@ -24,7 +35,7 @@ class AYONInterface(metaclass=_AYONInterfaceMeta):
     in the interface. By default, interface does not have any abstract parts.
     """
 
-    pass
+    log = None
```

Comment:
```suggestion
```

Reply:
this is actually helping with mypy and hits - `ITrayAddon` is using `self.log.warning()` but it is not defined there.